terraform/aks/config/staging.tfvars.json

```diff
@@ -5,7 +5,7 @@
     "infra_key_vault_name": "s189t01-gse-stg-inf-kv",
     "azure_enable_backup_storage": true,
     "key_vault_resource_group":  "s189t01-gse-stg-rg",
-    "enable_monitoring": false,
+    "enable_monitoring": true,
```

Comment:
can we change the statuscake website_url in this file to monitor /healthcheck.json instead of /check too?